README.md

```diff
@@ -101,17 +101,14 @@ Processors included in the collector:
 
 Once deployed to a Kubernetes cluster, the logs collection and processing configuration is stored as a ConfigMap under the `logs.config` key.
 
-To avoid processing an excessive amount of data, the `swo-k8s-collector` collects container logs only in `kube-*` namespaces, which means it only collects logs from the internal Kubernetes container. This behavior can be modified by setting `otel.logs.filter` value. An example for scraping logs from all namespaces:
+Collector by default gather all logs which migh be intensive. To avoid processing an excessive amount of data, the `swo-k8s-collector` can define filter which will drop all unwanted logs. This behavior can be modified by setting `otel.logs.filter` value. An example for scraping logs only from `kube-*` namespace:
```

Comment:
I would rather not change this until we release 4.x. This is what the users of 3.x will read, too, and it might be confusing.
Or, maybe even better, describe both behaviors here, the one in 3.x and the change in the upcoming 4.x.